flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/InternalTimer.java

```diff
@@ -23,8 +23,8 @@
 /**
  * Internal interface for in-flight timers.
  *
- * @param <K> type of the timer key.
- * @param <N> type of the timer namespace.
+ * @param <K> Type of the keys to which timers are scoped.	+ * @param <K> type of the timer key.
```

Comment:
Does this comment  mean `@param <K> Type of the keys to which timers are scoped.`?